test/tests/integration/probes/get-probes.test.ts

```diff
@@ -46,6 +46,7 @@ describe('Get Probes', () => {
 				.expect(200)
 				.expect((response) => {
 					expect(response.body).to.deep.equal([]);
+					expect(response).to.matchApiSchema();
```

Comment:
Only used here for now to test the integration. Once the other endpoints are documented, will be used in all tests.